cocinaborcelle/src/clientApi/carrinho/getCarrinho.ts

```diff
@@ -0,0 +1,11 @@
+import { Carrinho, Compra } from "@prisma/client";
+import api from "../api";
+
+export type CarrinhoCompleto = { carrinho: Carrinho & Compra };
```

Comment:
```suggestion
export type CarrinhoCompleto = {
	carrinho: Carrinho & { 
		compras: (Compra & { comida: Comida })[] 
	};
};
```

não esqueça que:
- está dando um include true pra comida também
- um carrinho tem um array de compras, no seu atributo .compras
- uma compra tem uma comida, no seu atributo .comida

para checar, é só passar o mouse por cima do que vc tá retornando lá na api do lado do servidor